### PR TITLE
[dnf5] Make ready and enable advisory/updateinfo tests for dnf5

### PR DESCRIPTION
**Pull request by kontura**
For: https://github.com/rpm-software-management/libdnf/pull/1534

### CODE REVIEW

**Comment by kontura** [comment]
Tests: https://github.com/rpm-software-management/ci-dnf-stack/pull/1101

**Comment by j-mracek** [comment]
I am not sure whether we can change the tests, because they are used for DNF4. I guess it requires a discussion with the team and QE

File: dnf-behave-tests/dnf/security-advisory.feature [hunk 1]
```diff
@@ -43,32 +47,56 @@ Scenario: upgrade cves
...
         | Action        | Package                   |
         | upgrade       | advisory_B-0:1.0-2.x86_64 |
 
 
-# @dnf5
-# TODO(nsella) Unknown argument "--sec-severity" for command "upgrade"
 Scenario: upgrade secseverity
-   When I execute dnf with args "upgrade --sec-severity Critical"
+   When I execute dnf4 with args "upgrade --sec-severity Critical"
+   When I execute dnf5 with args "upgrade --advisory-severities Critical"
```
Comment (j-mracek) [inline dnf-behave-tests/dnf/security-advisory.feature:70]: Have we agreed on a singular form of the options? `--advisory-severity`?

Comment (kontura) [reply dnf-behave-tests/dnf/security-advisory.feature:70]: We didn't, I chose the approach where options that accept a list have a plural form (`--advisory-severities=critical,moderate`, `--cves=CVE1,CVE2`, `--bzs=BZ1,BZ2..`,..) with some compatibility aliases such as: `--cve`, `--bz` - they are still available but not auto-completed.

Comment (j-mracek) [reply dnf-behave-tests/dnf/security-advisory.feature:70]: Ok, thanks for an explanation.

**Comment by kontura** [comment]
I have added more repoquery security filters tests.

**Comment by j-mracek** [comment]
LGTM

**Pull request merged**